lib/widgets/menuItem.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:flutter/material.dart';
+
+class MenuItem extends StatelessWidget {
+  final String title;
+  final Widget trailing;
+
+  MenuItem({this.title, this.trailing});
```

Comment:
Add onTap callback function here to be passed to ListTile child

Reply:
Fixed in 3c9d7409084cc8fa7e55cdacd96c2b96dc5e4606